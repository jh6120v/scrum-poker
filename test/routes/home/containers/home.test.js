import React from 'react';
import { render } from '@testing-library/react';
import Home from '../../../../src/routes/home/containers/home';

describe('test <Home />', () => {
    const { container, getByTestId } = render(<Home />);

    it('<Home /> snapshot', () => {
        expect(container.firstChild).toMatchSnapshot();
    });

    it('test <Home /> and find title', () => {
        const elem = getByTestId('display_title');

        expect(elem.textContent).toBe('Welcome to React-Redux-App!');
    });
});
