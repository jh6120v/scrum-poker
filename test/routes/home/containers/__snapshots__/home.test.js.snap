// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <Home /> <Home /> snapshot 1`] = `
<div
  class="sc-bdVaJa jggxrB"
>
  <div
    class="sc-bwzfXH bUbcxt"
  />
  <div
    class="sc-bxivhb kgXhAB"
  />
  <div
    class="sc-bwzfXH bUbcxt"
  />
  <h1
    class="sc-htpNat jUnqYe"
    data-testid="display_title"
  >
    Welcome to React-Redux-App!
  </h1>
</div>
`;
